core/cap-0038.md

```diff
@@ -452,10 +434,18 @@ index 75f39eb4..79c3995e 100644
      }
      body;
  };
-@@ -476,6 +536,15 @@ case ENVELOPE_TYPE_OP_ID:
+ 
+-union OperationID switch (EnvelopeType type)
++union HashIDPreImage switch (EnvelopeType type)
```

Comment:
I'm pretty sure preimage is typically one word, not two, so I believe it should be:
```suggestion
+union HashIDPreimage switch (EnvelopeType type)
```